dags/make_ciip_postgres_backup.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+# DAG to backup the ciip_portal postgres database every hour to a gcs bucket.
+"""
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
+from airflow.operators.python_operator import ShortCircuitOperator
+from airflow.contrib.kubernetes.secret import Secret
+from airflow.contrib.kubernetes.volume import Volume
+from airflow.contrib.kubernetes.volume_mount import VolumeMount
+from airflow.hooks.base_hook import BaseHook
+
+import os
+import json
+
+YESTERDAY = datetime.now() - timedelta(days=1)
+
+namespace = os.getenv('NAMESPACE')
+in_cluster = os.getenv('LOCAL_AIRFLOW', False) == 'False'
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': YESTERDAY,
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+}
+
+DAG_ID = os.path.basename(__file__).replace(".pyc", "").replace(".py", "")
+SCHEDULE_INTERVAL = '0 * * * *' # @hourly
+
+make_backup = DAG(DAG_ID, schedule_interval=SCHEDULE_INTERVAL, default_args=default_args)
+
+compute_resource = {'request_cpu': '1', 'request_memory': '2Gi', 'limit_cpu': '2', 'limit_memory': '4Gi'}
+
+cas_minio_connection = BaseHook.get_connection('cas_minio') # should I do this via minio?
```

Comment:
No, use gcs client directly. You can set the following env var: `    'GCS_KEY': json.loads(BaseHook.get_connection('cas_ggl_storage').extra)["extra__google_cloud_platform__keyfile_dict"]
`

Reply:
Oh I see you're doing that below already